src/validation/rules/StreamDirectiveOnListFieldRule.ts

```diff
@@ -24,6 +24,7 @@ export function StreamDirectiveOnListFieldRule(
         fieldDef &&
         parentType &&
         isDirective(GraphQLStreamDirective) &&
```

Comment:
Not totally sure why this is here or if it is needed.

Assuming there is a reason for it :+1:

Reply:
i think instead of this, i intended for a check to that the directive being validated is an instance of `GraphQLStreamDirective`